cmd/main.go

```diff
@@ -201,8 +201,32 @@ func (o options) Validate() error {
 }
 
 var defaultNoColor = func() bool {
-	if os.Getenv("GITHUB_ACTIONS") == "true" {
-		return false
+	// fatih/color will only output color when stdout is a terminal which is not
+	// true for many CI environments which support color output. So instead, we
+	// try to detect these CI environments via their environment variables.
+	// This code is based on https://github.com/jwalton/go-supportscolor
+	if _, exists := os.LookupEnv("CI"); exists {
+		var ciEnvNames = []string{
+			"APPVEYOR",
+			"BUILDKITE",
+			"CIRCLECI",
+			"DRONE",
+			"GITEA_ACTIONS",
+			"GITHUB_ACTIONS",
+			"GITLAB_CI",
+			"TRAVIS",
+		}
+		for _, ciEnvName := range ciEnvNames {
+			if _, exists := os.LookupEnv(ciEnvName); exists {
+				return false
+			}
+		}
+		if _, exists := os.LookupEnv("TEAMCITY_VERSION"); exists {
+			return false
+		}
```

Comment:
Could this be part of the `ciEnvNames` list above? Is there something different about this one that I'm missing?

Reply:
Ah, it should actually be outside the block that checks for `CI`, because that variable is likely not set as per this:

https://github.com/chalk/supports-color/blob/e06f284193939fbc1c75992871cda2b8192557ec/index.js#L114-L128

I'll fix that, thanks for noticing

Reply:
Fixed in https://github.com/gotestyourself/gotestsum/pull/330/commits/3e999571112b51972133b2b8981dd2143161f544.